app/models/concerns/bib_record.rb

```diff
@@ -89,6 +89,10 @@ def item_key
     fields.dig('item', 'key')
   end
 
+  def item_type
+    item.dig('itemType', 'key')
+  end
+
```

Comment:
```suggestion
```

This can be removed, see `item_type_code`

Reply:
oops, thought I got this deleted , thanks for catching it :)